packages/builder/test/builder.generate.test.js

```diff
@@ -25,8 +25,8 @@ describe('builder: builder generate', () => {
     r.mockImplementation((...args) => `r(${args.join(', ')})`)
     fs.readFile.mockImplementation((...args) => `readFile(${args.join(', ')})`)
     fs.outputFile.mockImplementation((...args) => `outputFile(${args.join(', ')})`)
-    jest.spyOn(path, 'join').mockImplementation((...args) => `join(${args.join(', ')})`)
-    jest.spyOn(path, 'resolve').mockImplementation((...args) => `resolve(${args.join(', ')})`)
+    jest.spyOn(path, 'join')
+    jest.spyOn(path, 'resolve')
```

Comment:
I had to prevent spies on this test. It generates artifacts like `r(foo, bar)` which are not comparable to check template being subdir of app dir